package.json

```diff
@@ -1,83 +1,32 @@
 {
-  "name": "docusign",
-  "version": "8.0.0",
-  "description": "A DocuSign API helper library with promise support",
-  "main": "docusign.js",
+  "name": "docusign-esign",
+  "version": "2.0.0",
+  "description": "DocuSign Node.js API client.",
+  "license": "LicenseRef-LICENSE",
+  "main": "src/index.js",
+  "author": "DocuSign Developer Center <devcenter@docusign.com>",
+  "contributors": [ "https://github.com/docusign/docusign-node-client/contributors" ],
+  "files": [ "src" ],
+  "keywords": [ "docusign", "esignature", "esign", "digital", "electronic", "transaction", "document", "certificate", "DTM", "PDF" ],
+  "engines": { "node": ">=4.2.0" },
+  "repository": { "type": "git", "url": "https://github.com/docusign/docusign-node-client.git" },
   "scripts": {
     "docs": "npm run update-docs && git add doc/ && git commit -m 'update docs' && npm run push-docs",
     "push-docs": "git subtree push --prefix doc origin gh-pages",
-    "update-docs": "jsdoc -d doc -r docusign.js dsUtils.js components/ README.md",
+    "update-docs": "jsdoc -d doc -r src/index.js src/ApiClient.js src/Configuration.js README.md",
     "lint": "semistandard",
-    "check-config": "./scripts/check-config.js",
-    "test-cb-examples": "mocha -t 30000 examples/callback-style",
-    "test-spec": "ava test/**/*.spec.js",
-    "test-cov": "nyc --reporter=lcov ava test/**/*.spec.js test/*.spec.js",
-    "test": "npm run lint && npm run test-cov"
+    "//check-config": "./scripts/check-config.js",
+    "//test": "./node_modules/mocha/bin/mocha SdkUnitTests"
```

Comment:
I have intentionally disabled tests temporarily, because DocuSign Security policy apparently doesn't allow posting integrator keys, even under Demo for test purposes. @Ergin008 is working with them to see if it's possible to use encrypted credentials for Travis CI & co. Until then automatic tests wont work, but devs could still use their creds if they wish to test the SDKs.